torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -2111,10 +2111,31 @@ Tensor symeig_backward(const std::vector<torch::autograd::Variable> &grads, cons
 
   auto vh = v.conj().transpose(-2, -1);
 
+  // Based on a little experiment, min_threshold should be
+  //   "max_eigenvalue * matrix_size * machine_eps"
+  // See: https://github.com/pytorch/pytorch/pull/50942
+  // However, as getting max_eigenvalue would need synchronization, I just take
+  //   a guess that the max eigenvalues would typically be less than 10
```

Comment:
Using the suggestion by @IvanYashchuk , you could compute exactly what you need and it will not require any synchronization as I see it.

Reply:
Thanks, did it in the new commit.